std_msgs/Header header

uint64 timestamp
uint64 timestamp_sample

float32[3] k_gains
float32[3] beta_gains
float32[3] lambda_gains
